Libplanet/Net/Transports/BoundPeerExtensions.cs

```diff
@@ -96,7 +96,7 @@ public static class BoundPeerExtensions
             {
                 Identity = Guid.NewGuid().ToByteArray(),
             };
-            var messageCodec = new MessageCodec();
+            var messageCodec = new TcpMessageCodec();
```

Comment:
Is a `BoundPeerExtensions` using `NetMQTransport`? If then, I guess we need to separate this class to `TCPBoundPeerExtensions`

Reply:
Eh, didn't think too much about it since it's only used for a console command. I agree, that the whole extension is rather problematic as `BoundPeerExtensions` is only for `TcpTransport` whereas `BoundPeer` class is agnostic of `ITransport` type. We can change the name to `TcpBoundPeerExtensions`, but either way, it is still problematic. ☹️

Reply:
Yeah, what I wanted to say more is, we need one pair of extensions if those extensions are dependent on some transport layer. In this case, For example, use `IMessageCodec` instead `TcpMessageCodec`.

Reply:
Huh... looking at the surrounding code again, there is already `QueryAppProtocolVersion()` for NetMQ and `QueryAppProtocolVersionTcp()` for TCP. I think #1612 should be resolved first for better interfacing this whole extension class. I'll change `QueryAppProtocolVersion()` to `QueryAppProtocolVersionNetMQ()` for now to at least somewhat mitigate the confusion.